coriolisclient/tests/cli/test_utils.py

```diff
@@ -0,0 +1,310 @@
+                "default": mock.sentinel.default_storage_backend,
+                "disk_mappings": mock.sentinel.disk_storage_mappings
+            },
+            result
+        )
+
+    def test_format_mapping(self):
+        mapping = {
+            "mapping1": "mock_mapping1",
+            "mapping2": "mock_mapping2",
+            "mapping3": "mock_mapping3"
+        }
+
+        result = utils.format_mapping(mapping)
+
+        self.assertEqual(
+            (
+                "'mapping1'='mock_mapping1', "
+                "'mapping2'='mock_mapping2', "
+                "'mapping3'='mock_mapping3'"
+            ),
+            result
+        )
+
+    def test_parse_storage_mappings(self):
+        storage_mappings = {
+            "default": "mock_default",
+            "backend_mappings": [
+                {
+                    "source": "mock_source",
+                    "destination": "mock_destination"
+                }
+            ],
+            "disk_mappings": [
+                {
+                    "disk_id": "mock_disk_id",
+                    "destination": "mock_destination"
+                }
+            ]
+        }
+
+        result = utils.parse_storage_mappings(storage_mappings)
+
+        self.assertEqual(
+            (
+                'mock_default',
+                {'mock_source': 'mock_destination'},
+                {'mock_disk_id': 'mock_destination'}
+            ),
+            result
+        )
+
+        result = utils.parse_storage_mappings(None)
+
+        self.assertEqual(
+            (None, {}, {}),
+            result
+        )
+
+    def test_format_json_for_object_property(self):
+        obj = mock.Mock()
+        obj.prop_name = {"key1": "value1", "key2": "value2"}
+
+        result = utils.format_json_for_object_property(obj, "prop_name")
+
+        self.assertEqual(
+            '{\n  "key1": "value1",\n  "key2": "value2"\n}',
+            result
+        )
+
+        obj = mock.Mock()
+        obj.prop_name.to_dict.return_value = \
+            {"key1": "value1", "key2": "value2"}
+
+        result = utils.format_json_for_object_property(obj, "prop_name")
+
+        self.assertEqual(
+            '{\n  "key1": "value1",\n  "key2": "value2"\n}',
+            result
+        )
+
+        obj.prop_name = None
+
+        result = utils.format_json_for_object_property(obj, "prop_name")
+
+        self.assertEqual(
+            '{}',
+            result
+        )
+
+    def test_validate_uuid_string(self):
+        result = utils.validate_uuid_string(
+            "12345678-9ABC-DEF1-2345-6789abcdef12")
+
+        self.assertEqual(
+            True,
+            result
+        )
+
+        result = utils.validate_uuid_string(
+            "123456789ABCDEF")
+
+        self.assertEqual(
+            False,
+            result
+        )
+
+    def test_add_args_for_json_option_to_parser(self):
+        parser = mock.Mock()
```

Comment:
I really loved the approach of using an empty `ArgumentParser` to test these kinds of methods. Can you please also do that here?